db/products.js

```diff
@@ -0,0 +1,69 @@
+let products = [];
```

Comment:
Same as message for articles about declaring arrays with `const` in this use case.